### DISCUSSION TITLE
Structurizr Lite Deployment Diagram

### DISCUSSION THREAD

**Discussion by Raphiniert2904** [structurizr/lite#142] [Q&A] [answered]
Hello everyone,

I found multiple examples to the Deployment Diagrams online, however I cant make them work for me. Some infomration of the diagrams is missing for me, and I dont know why, because all these examples seem to work.

For example, if I try to redo the official example for Deployment Diagrams, this is what I get

![Structurizr Example](https://github.com/structurizr/lite/assets/169131103/5dc4573b-b072-4cf2-88af-e06dda8329fb)

What missing is: Deployment Node, text at the rounded boxes. 

The Code I used is this right here: https://docs.structurizr.com/dsl/cookbook/deployment-groups/

Does anyone know, why the information is not shown?

**Comment by simonbrowndotje** [comment, marked as answer]
The example code at https://docs.structurizr.com/dsl/cookbook/deployment-groups/ doesn't have any styles configured, so my guess would be that you're additionally overriding the styles and the descriptive text is white on a white background.

**Comment by Raphiniert2904** [comment]
And you are absolutely right. Thanks alot. We really enjoy working with structurizr
